src/components/Embargo/Embargo.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Layout } from '@folio/stripes/components';
+
+export default class Embargo extends React.Component {
+    static propTypes = {
+      embargo: PropTypes.shape({
+        movingWallStart: PropTypes.shape({
+          length: PropTypes.number,
+          unit: PropTypes.string
+        }),
+        movingWallEnd: PropTypes.shape({
+          length: PropTypes.number,
+          unit: PropTypes.string
+        })
+      })
+    }
+
+    renderEmbargoEnd(embargo) {
+      if (embargo.movingWallEnd) {
+        const unitId = `ui-agreements.${embargo.movingWallEnd.unit}`;
+        const number = embargo.movingWallEnd.length;
+        return (
+          <Layout
+            data-test-embargo-end
+            data-test-embargo-end-length={number}
+            data-test-embargo-end-unit={embargo.movingWallEnd.unit}
+          >
+            <FormattedMessage id="ui-agreements.embargo.end" />
+            <FormattedMessage id={unitId} values={{ count: number }} />
+          </Layout>
+        );
+      }
+      return null;
+    }
+
+    renderEmbargoStart(embargo) {
+      if (embargo.movingWallStart) {
+        const unitId = `ui-agreements.${embargo.movingWallStart.unit}`;
+        const number = embargo.movingWallStart.length;
+        return (
+          <Layout
+            data-test-embargo-start
+            data-test-embargo-start-length={number}
+            data-test-embargo-start-unit={embargo.movingWallStart.unit}
+          >
+            <FormattedMessage id="ui-agreements.embargo.start" />
```

Comment:
Can you verify this works in RTL languages like Arabic? 

If it works correctly, it should show up something like `60 days Moving wall start:` (flipping the order of the two FormattedMessages).

Reply:
No, it shows like in English `Moving wall start: 60 days`
How can I fix that?

Reply:
[flexbox is a big help to us for RTL compatibility, particularly how it handles `flex-direction`](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). So I believe all you may have to do is set that `Layout`'s className as [`flex`](https://github.com/folio-org/stripes-components/blob/8d75fda73bf8a6bcda3d29a22234ea41a728534e/lib/Layout/styles/flexbox.css#L5). 

If you run into alignment issues, it may be because FormattedMessage renders `span`s by default and flexbox works best when its flex items are `div`s. Therefore, setting the `tagName` prop on them to `"div"` should solve it.

Reply:
setting `Layout` className to flex didn't solve the issue

Reply:
Yes, I tried that (although Mark mentioned that this should solve alignment issues) and it had no effect on the direction but broke the lines

Reply:
[This is basically what we're trying to accomplish here.](https://codepen.io/doytch/pen/PoPNdJj) There may be an element or something in this that's either getting erroneously added or missed. 

Note that if you're looking for it, `dir="rtl"` is applied directly on the wrapping element in that example. However, in Folio, it'll be applied on the top-level `html` tag.
<img width="1078" alt="Screen Shot 2020-04-17 at 8 59 20 AM" src="https://user-images.githubusercontent.com/5324374/79571834-dcb5d380-8089-11ea-89a5-ce8dea1410e5.png">

It may be easiest to debug this by writing a simplified case of just `div`s and getting that to flip its direction. Then, either replace the container with Layout or the items with the FormattedMessage. Then, put it all together. Step by step.

Reply:
![image](https://user-images.githubusercontent.com/8953703/79574290-f0633900-808d-11ea-9aa2-ae8151fbd587.png)
 I see this on the front-end when I just pass `tagName="div"` to the `FormattedMessage` s that we render in the `renderEmbargoEnd` and `renderEmbargoStart` functions....with a `className="flex"` on the containing `Layout`s

Reply:
Hm, thanks Aditya. I tried it and it didn't help. 
I tried it again today and it looks like in your example, I don't understand what's happened